.travis.yml

```diff
@@ -10,14 +10,21 @@ jdk:
 
 env:
   global:
-  - GPG_FILE_NAME="my.travis.gpg"
-  - ENCRYPTED_GPG_FILE_NAME="my.travis.gpg.enc"
-  - secure: OBEpztuhj8cQzssR5GgdfC6galSZuOAMHC6qgpjKyOepL+8WIRSaUyJnkmmCm4DLaQUAGVYBswRh0GeylYdakIkBdfjRWATTnyl7hgoTTV4DP2pMJ14DOg4FdhRCPKLXEoJjmk23/cKOixxezQtCEUGDJj3D6Q7S9SxvM6xhA7/Qgsnv8ClmkwmqB6R3Aa3uHQ0PSHJnWo1/CB8xW67CSv862QOTyoyHSH3uf+jCZLQifO//UiUf+GwOuyGHiqPjO7aRQJ2LHEIxu2Go1jPnYplKrma3myMgB220FQ7P5b+pPaQr9rsexM3Ksn9XcS1BMAIJ9DX+FJXK9iyYtmsJLz9vU+KaAa9Vz6NQVjKd64tuVd6xzbJ/oXhp9if3Ca9sBG8ZLwPZj4IarI1blYxCwlwh730rFDRER/TTMBDXq4cdtrK7IhAsNhpE0a0zZ00vxsVcSZFNrMJAFvD+L3RcOt9kpMPoZT/Q7Eoz+QF97ZiqzMBBhjskuZ83O6kk0hpudY1PPaOtggKLbICuLgPaOp2HNITy1hsU3QvjZkUSQfNbu4/lBP1Ae30IPeOQ1EtYfOA5nHsWioh9EYwuwpvZSBN2eZdf31+pXTpmiH77+xqbHsQ8dgovzvDxeE/OSsWIxISUr5kGIuyXENrgNghDBzBSE4xEe/lHWKVyy3gJcGo=
+  - GPG_FILE_NAME="ekino-gradle-java-plugin.gpg"
+  - ENCRYPTED_GPG_FILE_NAME="ekino-gradle-java-plugin.gpg.enc"
+  #PUBLISH_SNAPSHOT_REPO_URL
+  - secure: rfGTaiKC/iwQGN4BlpjPiyLM6QlZy6389oYew7loxfWfkC/6jvBBHdr6CyqUsoT7aUYMMqiZFzKI7lo+pZLPzu51KCdBiXJV7wlXtnKvTh3NeX2v3doJnOqk1FMhEaKNqwv73GKZOZZ6msbSXmqdnAZSb/i0AEHw9te3Wy3l0qURih2xlTJzk3Yio/B3EfRu6As0RjUvdlOToKmwqRNGINKlzaoFuJxMUbE8AlSHGwBliyyrem8MNLzqZy+5wkh07Hit/GcOGHSM0U0J9GngeQuabbVbKzmYt4LarWpJWtyfIcIOBSfQkQw/y9U008v6BC2kxEaTqX0qTMuHikap5euqgHszKv/rXpGnxDLOE8wjdIFLPX0jpRnsiOwFUdlKiGjYdFtGHihMc0VEbJjCYo7gZiKHM69s9omtugXjVysc3XVItgwRZ5WZV7jeD6Q7RybKy8cy3k1pUT8axmIGwgOHUEM1sSUSaCmciP7auk1za81v2hxa9L1jy5pd4L9iwEecAtqfZPSTLrxm5HNJgLRNiXc4yCq9udLY826FusENawOCK4YoJYrPVpbKuXyNeNcGGwwN9Bv1jiBGtkO2vQJaAAFUGw6PritkuSaZLcdwgt962WIdMBGQfMTh3GYp00B1ta+hH15kGw2wbcAgpiNa2kB5D/w0t89NUCNaiMI=
+  #PUBLISH_RELEASE_REPO_URL
   - secure: VRBPxcQl7Tfsd9MuTEMl1Rqd0xDm8BPfzDYkFHIwqoRn0kDy5r5UDILUmqiOcXwt57C92ykxtO4Kfo4Z8kReKjXbNyS+JPsScpiWefdNDDMdv3hoB81h/ZWSnr1NtEyZcMCDr1TVdV0UjOeap74YTqZMZVBI2j8mwph2cKowuDNAjoxZ7v5CocuU+TJYmOVASv/5Iy51HUgxVzDvk22QXbqfzBzcIECDq+YAgJxA5DwBmpzBjzSSC2HMQnJRWnm7cr9NvvFqAs9UDgGmq+iGj7AD95fn6yoDARUUJMm7wa0ZJVDoegUCLF0eHeKGazADHLYFwXPFNSJNdBv1f2DqokRFY5slxqVFyqIIngbtSZReJzncCXmXnoFi0h1E31eS9uJSu7zf6mrbqWvqGpIXgVCXd5epeC3krqmnEOrNfwNCY6rNjWScf6MfIDS6Gkau0ktrJtLtIQJNKUYpX6bPnJvkEkYD6d65uC+kjmOXZc8Eh5noVJoyBGJ7JP+AM4Sd/pm4MWvhVCM5dciOwj40M6DxWcEtBiRTwzeqn8jW4ZIENvGoUOu13wgk94/cU8LMRb/FaXzu8Vr0iaGfvcQpSiruFowyo4Vp9NoogGsHSXVM3rNud+wD2LwaayVB4f7g1GSA1VhLcY8JblW7SeJr+rrHvrmx8yRxsB8mMGY+y9E=
```

Comment:
Hi, I think we should keep urls in clear text. These are public ones and very easy to find.

Reply:
Right, let's do this to avoid future issues like this !